test/gl/gl_functions.test.cpp

```diff
@@ -1,3 +1,5 @@
+#if MLN_RENDER_BACKEND_OPENGL
```

Comment:
Maybe it's better to handle these at the build config level? Not sure.

Reply:
I tried that, but the `select` didn't seem to know about `//:metal_renderer` there.

Reply:
OK, I'll have a look at it sometime.